include/ostatus.php

```diff
@@ -897,13 +970,19 @@ private function completion($conversation_url, $uid, $item = array(), $self = ""
 				}
 
 				return($item_stored);
-			} else
+			} else {
 				return(-3);
+			}
 		}
 
 		$items = array_reverse($items);
 
 		$r = q("SELECT `nurl` FROM `contact` WHERE `uid` = %d AND `self`", intval($uid));
+
+		if (!dbm::is_result($r)) {
```

Comment:
A to-do doesn't help anyone here. I would rather prefer to comment out these added checks and to add a to-do as a reminder for the future to verify if they could be harmful under certain circumstances.